quickwit/quickwit-ingest/src/ingest_v2/shard_table.rs

```diff
@@ -24,46 +24,83 @@ use quickwit_proto::ingest::Shard;
 use quickwit_proto::types::SourceId;
 use quickwit_proto::IndexId;
 
-/// A set of open shards for a given index and source.
 #[derive(Debug, Default)]
-pub(super) struct ShardTableEntry {
+struct ShardTableEntryInfo {
```

Comment:
This new struct is not very useful, we can directly inline its attributes in `ShardTableEntry`:

```rust
pub(super) struct ShardTableEntry {
    local_shards: Vec<Shard>,
    local_round_robin_idx: AtomicUsize,
    remote_shards: Vec<Shard>,
    remote_round_robin_idx: AtomicUsize,
}
```